docker/pravega/scripts/init_controller.sh

```diff
@@ -10,6 +10,7 @@
 #
 
 init_controller() {
+    [ ! -z "$HOSTNAME" ] && add_system_property "config.controller.metricmetricsPrefix" "${HOSTNAME}"
```

Comment:
Is `metricmetricsPrefix` a typo, bug, or design choice?

Reply:
Is something that may need to be changes, as pointed out here: https://github.com/pravega/pravega/issues/3059